packages/builders/scripts/orchestration/init-stakeAtom.js

```diff
@@ -7,8 +7,16 @@ export const defaultProposalBuilder = async (
 ) => {
   const {
     hostConnectionId = 'connection-1',
-    controllerConnectionId = 'connection-0',
+    controllerConnectionId = 'connection-1',
     bondDenom = 'uatom',
+    bondDenomLocal = 'ibc/C4CFF46FD6DE35CA4CF4CE031E643C8FDC9BA4B99AE598E9B0ED98FE3A2319F9',
```

Comment:
it would be nice to get this from hashing... or to test that this matches what we get from hashing

Reply:
Agree. Something i think we'll tackle as part of #9211